core/src/main/java/org/stellar/anchor/auth/JwtService.java

```diff
@@ -50,25 +60,25 @@ public JwtService(
       String callbackAuthSecret,
       String platformAuthSecret,
       String custodyAuthSecret) {
-    this.sep10JwtSecret = toBase64OrNull(sep10JwtSecret);
-    this.sep24InteractiveUrlJwtSecret = toBase64OrNull(sep24InteractiveUrlJwtSecret);
-    this.sep24MoreInfoUrlJwtSecret = toBase64OrNull(sep24MoreInfoUrlJwtSecret);
-    this.callbackAuthSecret = toBase64OrNull(callbackAuthSecret);
-    this.platformAuthSecret = toBase64OrNull(platformAuthSecret);
-    this.custodyAuthSecret = toBase64OrNull(custodyAuthSecret);
+    this.sep10JwtSecret = toSecretKeySpecOrNull(sep10JwtSecret);
+    this.sep24InteractiveUrlJwtSecret = toSecretKeySpecOrNull(sep24InteractiveUrlJwtSecret);
+    this.sep24MoreInfoUrlJwtSecret = toSecretKeySpecOrNull(sep24MoreInfoUrlJwtSecret);
+    this.callbackAuthSecret = toSecretKeySpecOrNull(callbackAuthSecret);
```

Comment:
Is it necessary to have the secret?

Reply:
Sorry what do you mean?

Reply:
The way we use JWT for SEP10 and SEP24 do not seem to require a fixed secret. Is it possible to generate a random secret so that we can further simplify the secret configuration.

Reply:
SEP-10 and SEP-24 do require fixed JWT. Otherwise, is we generate it randomly, tokens issued between the starts will not be valid. 
However, I like this idea and I think we can implement it in v3 (for anchors that are fine with tokens being invalid between restarts)

Reply:
I think we need the SEP-24 JWT secret to be configured by the business so that they can validate the signature before authenticating the user in their webview, but the SEP-10 JWT secret could be internalized since the business never needs to validate the SEP-10 signature itself.